src/biokbase/narrative/jobs/job.py

```diff
@@ -175,43 +157,45 @@ def __getattr__(self, name):
         Map expected job attributes to paths in stored ee2 state
         """
         attr = dict(
-            app_id=lambda: self._const_state.get("job_input", {}).get(
+            app_id=lambda: self._acc_state.get("job_input", {}).get(
                 "app_id", JOB_ATTR_DEFAULTS["app_id"]
             ),
-            app_version=lambda: self._const_state.get("job_input", {}).get(
+            app_version=lambda: self._acc_state.get("job_input", {}).get(
                 "service_ver", JOB_ATTR_DEFAULTS["app_version"]
             ),
             batch_id=lambda: (
                 self.job_id
                 if self.batch_job
-                else self._const_state.get("batch_id", JOB_ATTR_DEFAULTS["batch_id"])
+                else self._acc_state.get("batch_id", JOB_ATTR_DEFAULTS["batch_id"])
             ),
-            batch_job=lambda: self._const_state.get(
+            batch_job=lambda: self._acc_state.get(
                 "batch_job", JOB_ATTR_DEFAULTS["batch_job"]
             ),
-            cell_id=lambda: self._const_state.get("job_input", {})
+            cell_id=lambda: self._acc_state.get("job_input", {})
             .get("narrative_cell_info", {})
             .get("cell_id", JOB_ATTR_DEFAULTS["cell_id"]),
-            child_jobs=lambda: self.state().get(
```

Comment:
Should attributes `child_jobs` and `retry_ids` cause a EE2 `check_job` when accessed? ... need to look into this

Reply:
Yes, I think so -- they change during the course of the job's existence.

Reply:
I'm holding off on this because putting a `state()` call in `__getattr__` turns out to make issues during instantiation and generally complicates things

Reply:
We are going to need it at some point. I remember when I was looking at this it was very easy to start introducing circular logic. 😖